jsonp-policy/src/main/java/io/apiman/plugins/jsonp_policy/http/HttpHeaders.java

```diff
@@ -0,0 +1,52 @@
+package io.apiman.plugins.jsonp_policy.http;
+
+import java.util.Map;
+
+/**
+ * Utility class to get and set data into headers.
+ *  
+ * @author Alexandre Kieling <alex.kieling@gmail.com>
+ */
+public class HttpHeaders {
```

Comment:
Nice.  Perhaps ServiceRequest and ServiceResponse should have something similar to this class instead of a simple map.